src/extension/importer/rest/src/main/java/org/geoserver/importer/rest/ImportJSONReader.java

```diff
@@ -281,12 +286,37 @@ ImportTransform transform(JSONObject json) throws IOException {
             catch(Exception e) {
                 throw new ValidationException("Error parsing reproject transform", e);
             }
+        } else if ("GdalTranslateTransform".equalsIgnoreCase(type)) {
```

Comment:
Some ability to register / read new transformers in a more dynamic way here is probably advisable in the long term, but is probably out of the context of this work.